frontend/src/services/data.service.js

```diff
@@ -1,97 +1,38 @@
 import axios from "axios";
-import Cookies from "universal-cookie";
-import * as jwt from "jsonwebtoken";
-
-const cookies = new Cookies();
-const baseUrl =
-    process.env.NODE_ENV === "development"
-        ? "http://localhost:5000/api"
-        : "https://suggestit.chalmers.it/api";
-const authCookieName = "AUTH_TOKEN";
 
 export const updateSuggestions = setter =>
     axios
-        .get(`${baseUrl}/`, {
-            headers: {
-                Authorization: cookies.get(authCookieName),
-            },
-        })
+        .get(`/api`)
         .then(res => {
             setter(res.data ? res.data : []);
         })
         .catch(err => {
             //User might not be logged in
         });
 
-export const getSuggestion = uuid =>
-    axios.get(`${baseUrl}/suggestion?Id=${uuid}`, {
-        headers: {
-            Authorization: cookies.get(authCookieName),
-        },
-    });
+export const getSuggestion = uuid => axios.get(`/api/suggestion?Id=${uuid}`);
 
-export const addSuggestion = _suggestion =>
-    axios.post(`${baseUrl}/`, _suggestion);
+export const addSuggestion = _suggestion => axios.post(`/api/`, _suggestion);
 
 export const deleteSuggestion = uuid =>
-    axios
-        .delete(`${baseUrl}/delete?Id=${uuid}`, {
-            headers: {
-                Authorization: cookies.get(authCookieName),
-            },
-        })
-        .catch(error => {
-            //User might not be logged in
-        });
+    axios.delete(`/api/delete?Id=${uuid}`).catch(error => {
+        //User might not be logged in
+    });
 
 export const deleteSuggestions = suggestions =>
-    axios.put(
-        `${baseUrl}/delete`,
-        {
-            ids: suggestions.map(e => e.id),
-        },
-        {
-            headers: {
-                Authorization: cookies.get(authCookieName),
-            },
-        }
-    );
+    axios.put(`/api/delete`, {
+        ids: suggestions.map(e => e.id),
+    });
 
-export const loginRedirect = () =>
-    axios
-        .get(`${baseUrl}/clientid`)
-        .then(res =>
-            window.location.replace(
-                `https://ldap-auth.chalmers.it/authenticate?client_id=${res.data.client_id}`
-            )
-        )
-        .catch(err => console.log("Failed to get client_id"));
+export const loginRedirect = () => (window.location = "/api/login");
```

Comment:
Is this redirect to Gamma?

Reply:
Should it not be `gamma.chalmers.it` ?

Reply:
The backend handles the redirect to gamma.
The backend redirects to the backend which redirects to gamma since you cannot set environment variables for production version of suggestit frontend.